redhat-operators/tektoncd-pipeline-operator/tektoncd-pipeline-operator.v0.0.1.clusterserviceversion.yaml

```diff
@@ -0,0 +1,248 @@
+    mediatype: image/png
+  links:
+  - name: Tekton Pipeline GitHub Repository
+    url: https://github.com/tektoncd/pipeline
+  - name: Tekton Pipeline Operator GitHub Repository
+    url: https://github.com/openshift/tektoncd-pipeline-operator
+  keywords: ['tektoncd', 'build', 'pipeline']
+  apiservicedefinitions: {}
+  customresourcedefinitions:
+    owned:
+    - name: installs.tekton.dev
+      version: v1alpha1
+      kind: Install
+      displayName: TektonCD-Pipeline Install
+      description: CRD to trigger install/delete of TektonCD-Pipeline
+      resources:
+      - kind: Pod
+        version: v1
+      - kind: Service
+        version: v1
+      - kind: ServiceAccount
+        version: v1
+      - kind: ConfigMap
+        version: v1
+      specDescriptors: []
+      statusDescriptors:
+      - description: List of resources installed by the operator for this cr
+        displayName: Installed Resources
+        path: resources
+      - description: Version of Tektoncd Pipeline installed
+        displayName: Tektoncd Pipeline Version
+        path: version
+  description: Kubernetes operator to install TektonCD-Pipeline
+  displayName: TektonCD Pipeline Operator
+  install:
+    spec:
+      clusterPermissions:
```

Comment:
Does this operator require clusterPermissions to function properly in a SingleNamespace installMode?
When using a `permissions` section in the CSV, if the CSV is installed with the AllNamespaces installMode, OLM will upgrade the permissions to clusterPermissions automatically.

Reply:
I see. I shall test locally and update it

Reply:
yes, it does. This operator creates cluster permissions for openshift-pipe lines. So taht openshift pipelines can watch `taskruns` `pipelineruns` etc created in any namespace.

At present `AllNamespaces` installMode doesn't fit our purpose. So we have set `AllNamespaces` installMode to false.